cmd/kops/create_cluster.go

```diff
@@ -124,8 +126,17 @@ type CreateClusterOptions struct {
 
 	// ConfigBase is the location where we will store the configuration, it defaults to the state store
 	ConfigBase string
+
+	// DryRun we output data
+	DryRun bool
+
+	// Output type during a DryRun
+	Output string
 }
 
+var targetOptions = sets.NewString(cloudup.TargetDryRun, cloudup.TargetCloudformation, cloudup.TargetDirect, cloudup.TargetTerraform)
```

Comment:
I don't think dryrun should be an explicit target here in the CLI.

Reply:
correct ... will remove